frontend/src/components/pages/Home/Home.js

```diff
@@ -5,13 +5,24 @@ class Home extends React.Component {
     render = () => {
         return (
             <>
-                <ul>
-                    <li><Link to="/create-room">Create room</Link></li>
-                    <li><Link to="/join-room">Join room</Link></li>
-                </ul>
+                <div className="label-title">LE QUIZ
+                </div>
+                <div className="home-menu">
+                    <div className="home-menu-wrapper">
+                        <button className="btn-create-room btn-menu" onClick={() => document.location.href = "/create-room"}>
+                            <p className="home-menu-label">Créer un Salon</p>
+                        </button>
+                    </div>
+                    <div className="home-menu-wrapper">
+                        <button className="btn-join-room btn-menu" onClick={() => document.location.href = "/join-room"}>
```

Comment:
It's not the way we redirect users in the app, you should use the history object